state/runtime/evm/instructions.go

```diff
@@ -907,6 +907,11 @@ func opGasLimit(c *state) {
 	c.push1().SetInt64(c.host.GetTxContext().GasLimit)
 }
 
+func opBaseFee(c *state) {
+	baseFee := big.NewInt(0).Set(c.host.GetTxContext().BaseFee)
```

Comment:
maybe `baseFee := new(big.Int).Set(c.host.GetTxContext().BaseFee)` is better because there is only one initialization?

Reply:
In order to fix failing test I think you should I add something like:
```
func opBaseFee(c *state) {
    if !c.config.London {
        c.exit(errOpCodeNotFound)

        return
    }

    baseFee := new(big.Int).Set(c.host.GetTxContext().BaseFee)
    c.push1().SetUint64(baseFee.Uint64())
}
```

Reply:
Done